scm-webapp/src/main/java/sonia/scm/api/v2/resources/MeResource.java

```diff
@@ -139,9 +135,4 @@ public Response changePassword(@Valid PasswordChangeDto passwordChange) {
     );
     return Response.noContent().build();
   }
-
-  @Path("api_keys")
-  public ApiKeyResource apiKeys() {
-    return apiKeyResource.get();
-  }
```

Comment:
This is a breaking change. You cannot delete endpoints from our rest api. Some users like plugin developers could be using this...